pkg/tracer/tcptracer-ebpf.go

```diff
@@ -0,0 +1,235 @@
+// Code generated by go-bindata.
```

Comment:
But then you will need to recommit it every time the C-code is modified, right? Will patches be reasonable? (i.e. if only the `_tcptracerEbpfO` changes, then that's OK)